src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildStaticGraphRestore.cs

```diff
@@ -716,6 +736,7 @@ internal static List<TargetFrameworkInformation> GetTargetFrameworkInfos(IReadOn
                 }
 
                 var dependencies = GetPackageReferences(msBuildProjectInstance, isCpvmEnabled, centralPackageVersions);
+                var prunedReferences = msBuildProjectInstance.IsPropertyTrue("RestoreEnablePackagePruning") ? GetPrunePackageReferences(msBuildProjectInstance) : [];
```

Comment:
Is there a string const for "RestoreEnablePackagePruning" ?

Reply:
Not one available here. 
We don't really have a properties const for this in the CLI codepaths. 
We probably should, but in this case I followed the established pattern.

We can create a tracking issue but I doubt it's something we need tracked. It's one of those, refactor when someone has time. 

lmk what you think we should do.

Reply:
Okay just wanted to make sure, if we have a const use it, if not, I don't think they are always necessary.